site/_includes/timeline.njk

```diff
@@ -0,0 +1,110 @@
+<div class="feature-timeline" id="{{ feature.id }}-timeline">
+</div>
+<script type="module" defer>
+  const featureTimelineContainer = document.getElementById("{{ feature.id }}-timeline");
+
+  const featureData = [
+    {% if feature.status.baseline === "low" %}
+      {
+        "event": "low",
+        "date": new Date("{{ feature.status.baseline_low_date }}"),
+        "datestring": "{{ feature.status.baseline_low_date }}"
+      },
+    {% endif %}
+    {% if feature.status.baseline === "high" %}
+      {
+        "event": "high",
+        "date": new Date("{{ feature.status.baseline_high_date }}"),
+        "datestring": "{{ feature.status.baseline_high_date }}"
+      },
+    {% endif %}
+    {% for browser in browsers %}
+      {% set version = feature.status.support[browser.id] %}
+      {% if version %}
+        {
+          "event": "{{ browser.id }}",
+          "date": new Date("{{ browser.releases[version].release_date }}"),
```

Comment:
This will need fixing with the recent change where certain versions can contain the `≤` character. For example the `sub-sup` feature.

Reply:
See #acafc29353b4180a52c04ee3206f2f453e809152.